content/news/2025-12-15-fq2bam/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "GPU-accelerated genome mapper Parabricks FQ2BAM is now in Galaxy"
+date: "2025-12-15"
+authors: Anup Kumar 
+tease: "NVIDIA Parabricks' GPU-accelerated FQ2BAM is now available in Galaxy, bringing FASTQ-to-BAM alignment runtime to just a few minutes for 2.5 GB paired-end human whole-genome datasets. Its runtime is benchmarked against BWA-MEM2 and BWA across five runs and shows that mapping and QC metrics remain essentially identical to CPU-based alignment—while delivering a major speedup for Galaxy workflows."
+subsites: [global,eu,us,freiburg]
+---
+
+# **Parabricks FQ2BAM mapper**
+
+NVIDIA's [Parabricks](https://docs.nvidia.com/clara/parabricks/latest/gettingstarted/gettingthesoftware.html#running-nvidia-parabricks) developed a GPU-accelerated tool suite for genomic analysis. One of the tools from the suite is [FQ2BAM](https://usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fparabricks_fq2bam%2Fparabricks_fq2bam%2F4.6.0-1%2Bgalaxy0&version=latest) which accelerates mapping of DNA reads. It is now integrated into Galaxy and runs on an L40S GPU to generate read alignments in a few minutes for zipped paired-end FASTQ reads of size as large as 2.5 GB. It is a drop-in replacement for BWA-MEM2 while maintaining the same mapping quality. [FQ2BAM](https://docs.nvidia.com/clara/parabricks/latest/documentation/tooldocs/man_fq2bam.html) tool wraps BWA-MEM2 to perform alignment from FASTQ to BAM and then handles downstream processing such as sorting, with optional steps for duplicate marking and base-quality score recalibration.
```

Comment:
> of size as large as 2.5 GB

Sounds weird - like bad things happening when you go over that size?

Reply:
fixed it :)